raptorq/node/grpc/client.go

```diff
@@ -12,17 +13,22 @@ import (
 )
 
 const (
-	logPrefix = "raptorqClient"
+	logPrefix             = "raptorqClient"
+	defaultConnectTimeout = 5 * time.Second
 )
 
 type client struct{}
 
 // Connect implements node.Client.Connect()
 func (client *client) Connect(ctx context.Context, address string) (node.Connection, error) {
+	// Limits the dial timeout, prevent got stuck too long
+	dialCtx, cancel := context.WithTimeout(ctx, defaultConnectTimeout)
+	defer cancel()
```

Comment:
should add a timeout context here to prevent got stuck here too long in case rqservice is not available.